app/todos/resources.py

```diff
@@ -0,0 +1,85 @@
+# Ask about getting branches up to date once committed
```

Comment:
Hopefully I answered this on Friday :), but if you need a reminder, the general process is:
1) Bring you local master branch up to date `git pull origin master`
2) Merge your master branch to your working branch `git merge master`
3) Resolve any conflicts that pop up.
4) Add and commit